build/broccoli/build-packages.js

```diff
@@ -74,7 +74,8 @@ function topsortPackages() {
 function treeForPackage(packagePath) {
   let packageTree = buildPackage({
     projectPath: packagePath,
-    tsconfigPath: TSCONFIG_PATH
+    tsconfigPath: TSCONFIG_PATH,
+    include: ['lib/**/*.ts', 'index.ts']
```

Comment:
@stefanpenner But this line is needed because `@glimmer/build` defaults to `['src/**/*.ts']`. I think it's passed to the tsconfig and not necessarily Broccoli, although I'd need to double check.